releases.md

```diff
@@ -275,6 +276,7 @@ Older releases are EOL and available on [GitHub][tekton-pipeline-releases].
 [v0-37-0]: https://github.com/tektoncd/pipeline/releases/tag/v0.37.0
 
 [v0.60-0-docs]: https://github.com/tektoncd/pipeline/tree/v0.60.0rdocs#tekton-pipelines
```

Comment:
nit: Just realized `r/ should be replaced by '/'`

Reply:
Fixed